src/quemb/molbe/autofrag.py

```diff
@@ -1,12 +1,221 @@
-# Author: Oinam Romesh Meitei
-
+# Author: Oinam Romesh Meitei, Shaun Weatherly
 
+import networkx as nx  # type: ignore
+import numpy as np
 from numpy.linalg import norm
+from pyscf import gto
 
 from quemb.molbe.helper import get_core
 from quemb.shared.helper import unused
 
 
+def euclidean_norm(
+    i_coord: float,
+    j_coord: float,
+):
```

Comment:
I don't want to force type-hinting, but if you use it, then I would also add the return type.
i.e. `-> float` in this case.

This comment applies to the rest of the code